USFM_Converter/MainForm.cs

```diff
@@ -517,11 +464,55 @@ private HTMLConfig BuildConfig()
             {
                 config.divClasses.Add(TextAlignmentClasses[3]);
             }
+            config.divClasses.Add(fontClass);
+
+            // Will be added to HTML config class 
+            config.separateVerses = willSeparateVerse;
+
             config.separateChapters = willSeparateChap;
+
             return config;
         }
+        private void setColorFocus(object sender,bool focus)

```

Comment:
If this is going to be always a button passed in then send in a button instead of casting it to button. Would make things a whole lot simpler

Reply:
Oh yeah you're right!